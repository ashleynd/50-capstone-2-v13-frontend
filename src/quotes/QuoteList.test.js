import React from "react";
import { render } from "@testing-library/react";
import Quotes from "./QuoteList";

it("renders without crashing", function() {
  render(<Quotes />);
});

it("matches snapshot with no quotes", function() {
  const { asFragment } = render(<Quotes />);
  expect(asFragment()).toMatchSnapshot();
});
