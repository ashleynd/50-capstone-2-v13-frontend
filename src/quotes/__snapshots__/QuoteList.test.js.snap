// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="LoadingSpinner"
  >
    <img
      alt="spinner"
      src="spinner.gif"
      width="150px"
    />
  </div>
</DocumentFragment>
`;

exports[`matches snapshot with no quotes 1`] = `
<DocumentFragment>
  <div
    class="LoadingSpinner"
  >
    <img
      alt="spinner"
      src="spinner.gif"
      width="150px"
    />
  </div>
</DocumentFragment>
`;
